src/content/docs/data-apis/custom-data/aws-privatelink.mdx

```diff
@@ -512,9 +564,36 @@ These are the New Relic endpoint services available via AWS PrivateLink:
             `com.amazonaws.vpce.us-west-2.vpce-svc-06192ca8948a1e41c`
           </td>
         </tr>
+        <tr>
+          <td>
+            New Relic REST API
+          </td>
+          <td>
+            `api.newrelic.com`
+          </td>
+          <td>
+            `com.amazonaws.vpce.us-west-2.vpce-svc-0e8e1c3c0fd16ff9c`
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            Synthetics job manager
+          </td>
+          <td>
+            `synthetics-horde.nr-data.net`
+          </td>
+          <td>
+            `com.amazonaws.vpce.us-west-2.vpce-svc-0e8e1c3c0fd16ff9c`
+          </td>
+        </tr>
 
         </tbody>
       </table>
+      <Callout variant="important">
+        Review the following constraints when configuring the `api.newrelic.com`, `identity-api.newrelic.com`, `infrastructure-command-api.newrelic.com` or `synthetics-horde.nr-data.net` hostnames:
+        - The endpoint service does not have an associated DNS private name. Create a PrivateLink connected to this service endpoint, and create the Private Hosted Zone (PHZ) for each hostname.
```

Comment:
```suggestion
        The endpoint services for `api.newrelic.com`, `identity-api.newrelic.com`, `infrastructure-command-api.newrelic.com`, and `synthetics-horde.nr-data.net` do not have associated DNS private names.
        * To configure access to these services, create a PrivateLink connection to each service endpoint.
        * For each hostname listed above, create a corresponding Private Hosted Zone (PHZ) within your VPC.
```

Reply:
I'm down with that. I just copied the existing language but I'm happy to update it.